Automattic/wp-calypso
# Remove all Traffic Guide related code

#### Proposed Changes

Context: pau2Xa-4d3-p2

* Removes all Traffic Guide related code.
* Redirects `/marketing/ultimate-traffic-guide` to `https://wpcourses.com/course/intro-to-search-engine-optimization-seo/`

#### Testing Instructions



* Go to `/marketing/ultimate-traffic-guide` and confirm that you are redirected to `https://wpcourses.com/course/intro-to-search-engine-optimization-seo/`
* Confirm that the thank you page works correctly by purchasing a domain.

#### Pre-merge Checklist



- [x] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes? (Removed tests)
